knora/dsplib/models/sipi.py

```diff
@@ -45,5 +44,5 @@ def upload_bitstream(self, filepath):
             req = requests.post(self.sipi_server + "/upload?token=" + self.token, files=files)
         on_api_error(req)
         print(f'Uploaded file {filepath}')
-        res = req.json()
+        res: dict[Any, Any] = req.json()
```

Comment:
Seems weird to me to type this, when we use "Any" anyways.

Reply:
This helps in xml_upload.py, where `img = sipi_server.upload_bitstream(os.path.join(imgdir, resource.bitstream.value))` is then automatically recognized as dict. It's still quite imprecise, but better than Any, in my opinion.